src/target/nxpke04.c

```diff
@@ -191,7 +191,7 @@ bool ke04_probe(target_s *t)
 	case SRSID_PIN_80: {
 		/* We have either a KE04Z64 or 128 */
 		/* Try to read a flash address not available in a Z64 */
-		volatile uint32_t __attribute__((unused)) dummy = target_mem_read32(t, 0x00010000U);
+		volatile uint32_t UNUSED dummy = target_mem_read32(t, 0x00010000U);
```

Comment:
This macro needs defining in general.h so it's visible to this code please. It should probably be prefixed `BMD_` as in `BMD_UNUSED` for naming consistency.